src/app/shared/components/forecast-card/forecastCard.controller.js

```diff
@@ -23,6 +23,8 @@ export default class ForecastCardController {
             targetEvent: event,
             clickOutsideToClose: true,
             escapeToClose: true
+        }).catch(() => {
```

Comment:
If you are not doing anything here then you don't have to put the catch, if you put the catch then you have to do something

Reply:
Hm, I did this because if I didn't put a catch it throws an error of "possibly unhandled exception"